content/kotlin/concepts/datetime/terms/todayIn/todayIn.md

```diff
@@ -0,0 +1,43 @@
+---
+Title: '.todayIn()'
+Description: 'Passing a specific time zone as an argument and calling this method on an Instant value of Clock will return a date.'
+Subjects:
+  - 'Computer Science'
+  - 'Code Foundations'
+Tags:
+  - 'Methods'
+  - 'Date'
+  - 'Kotlin'
+CatalogContent:
+  - 'learn-kotlin'
+  - 'paths/computer-science'
+---
+
+The **`.todayIn()`** method returns a date when called on an Instant value of `Clock` while passing a `TimeZone` argument.
+
+## Syntax
+
+```pseudo
+Clock.System.todayIn(timeZone: TimeZone.currentSystemDefault())
```

Comment:
```suggestion
myClock.todayIn(myTimeZone)
```
The Syntax example needs to be more generic.  (It can be any Clock and any TimeZone)